.github/workflows/verify-release.yml

```diff
@@ -75,7 +75,7 @@ jobs:
         # Expression expands to ["ubuntu-latest","windows-latest"] or ["ubuntu-latest","windows-latest","macos-latest"]
         # GitHub expressions don't have 'if' statements, so we use a ternary operator to conditionally include the MacOS runner suffix.
         # See the docs for a similar example to this: https://docs.github.com/en/actions/learn-github-actions/expressions#fromjson
-        runner: ${{ fromJSON(format('["ubuntu-latest","windows-latest"{0}]', github.event.inputs.enableMacRunner == 'true' && ',"macos-latest"' || '')) }}
+        runner: ${{ fromJSON(format('["ubuntu-latest","windows-latest"{0}]', github.event.inputs.enableMacRunner && ',"macos-latest"' || '')) }} #
```

Comment:
I figured this out the other day, you can test this by executing the action pointing to this branch

![Screenshot 2025-01-16 at 1 10 46 PM](https://github.com/user-attachments/assets/d6128863-2ca3-47ee-9ace-82ca8e2b02ea)